mta-sts.md

```diff
@@ -198,32 +194,78 @@ This JSON object contains the following key/value pairs:
 * `mx`: MX identity patterns (list of plain-text strings, required). One or more
   patterns matching a Common Name ([@!RFC6125]) or Subject Alternative Name
   ([@!RFC5280]) DNS-ID present in the X.509 certificate presented by any MX
-  receiving mail for this domain.  For example, `["mail.example.com",
-  ".example.net"]` indicates that mail for this domain might be handled by any
+  receiving mail for this domain.  For example:
+  ```
+   mx: mail.example.com
+   mx: .example.net
+  ```
+  indicates that mail for this domain might be handled by any
   MX with a certificate valid for a host at `mail.example.com` or `example.net`.
   Valid patterns can be either fully specified names (`example.com`) or suffixes
   (`.example.net`) matching the right-hand parts of a server's identity; the
-  latter case are distinguished by a leading period.  In the case of
-  Internationalized Domain Names ([@!RFC5891]), the MX MUST specify the
-  Punycode-encoded A-label [@!RFC3492] and not the Unicode-encoded U-label. The
-  full semantics of certificate validation are described in
-  (#mx-certificate-validation), "MX Certificate Validation."
+  latter case are distinguished by a leading period.  If there are more than
+  one MX specified by the policy, they MUST be on separate lines within the
+  policy file.  In the case of Internationalized Domain Names ([@!RFC5891]), 
+  the MX MUST specify the Punycode-encoded A-label [@!RFC3492] and not the 
+  Unicode-encoded U-label. The full semantics of certificate validation are 
+  described in (#mx-certificate-validation), "MX Certificate Validation."
 
-An example JSON policy is as below:
+An example policy is as below:
 
 ```
-{
-  "version": "STSv1",
-  "mode": "enforce",
-  "mx": [".mail.example.com"],
-  "max_age": 123456
-}
+version: STSv1
+mode: enforce
+mx: mail.example.com
+mx: .example.net
+mx: backupmx.example.com
+max_age: 123456
 ```
 
+The formal definition of the policy resource, defined using [@!RFC7405], is as
+follows:
+
+    sts-policy-record        = sts-policy-version CRLF
+                               sts-policy-mode CRLF
+                               1*(sts-policy-mx CRLF)
+                               sts-policy-max-age
+
+    field-delim              = ":" *WSP
```

Comment:
Would it be better to have at least one space between ":" and value?

Replace:
field-delim              = ":" *WSP
with 
field-delim              = ":" 1*WSP